game/plugin/skills/fishing/src/fishing_data.kt

```diff
@@ -0,0 +1,59 @@
+package org.apollo.game.plugin.skills.fishing
+
+import org.apollo.game.model.Animation
+import org.apollo.game.model.Position
+import java.util.*
+import  org.apollo.game.plugin.skills.fishing.Fish.*
+import org.apollo.game.plugin.skills.fishing.FishingTool.*
+
+
+enum class Fish(val id: Int, val level: Int, val experiance: Double) {
+    SHRIMP(317, 1, 10.0),
+    SARDINE(327, 5, 20.0),
+    MACKEREL(353, 16, 20.0),
+    HERRING(345, 10, 30.0),
+    ANCHOVY(321, 15, 40.0),
+    TROUT(335, 20, 50.0),
+    COD(341, 23, 45.0),
+    PIKE(349, 25, 60.0),
+    SALMON(331, 30, 70.0),
+    TUNA(359, 35, 80.0),
+    LOBSTER(377, 40, 90.0),
+    BASS(363, 46, 100.0),
+    SWORDFISH(371, 50, 100.0),
+    SHARK(383, 76, 110.0),
+}
+
+enum class FishingTool(val id: Int, val animation: Animation, val message: String, val bait: Int) {
+    LOBSTER_CAGE(301, Animation(619), "You attempt to catch a lobster...", -1),
+    SMALL_NET(303, Animation(620),  "You cast out your net...", -1),
+    BIG_NET(305, Animation(620),  "You cast out your net...", -1),
+    HARPOON(311, Animation(618),  "You start harpooning fish...", -1),
+    FISHING_ROD(307, Animation(622),  "You attempt to catch a fish...", 313),
+    FLY_FISHING_ROD(309, Animation(622),  "You attempt to catch a fish...", 314),
+}
+
+enum class FishingSpot(val id: Int, val primaryTool: FishingTool, val secondaryTool: FishingTool, val primaryFish: Array<Fish>, val secondaryFish: Array<Fish>) {
+    ROD(309, FISHING_ROD, FLY_FISHING_ROD, arrayOf<Fish>(TROUT, SALMON), arrayOf<Fish>(PIKE)),
+    CAGE_HARPOON(312, LOBSTER_CAGE, HARPOON, arrayOf<Fish>(LOBSTER), arrayOf<Fish>(TUNA, SWORDFISH)),
+    NET_HARPOON(313, BIG_NET, HARPOON, arrayOf<Fish>(MACKEREL, COD), arrayOf<Fish>(BASS, SHARK)),
+    NET_ROD(316, SMALL_NET, FISHING_ROD, arrayOf<Fish>(SHRIMP, ANCHOVY), arrayOf<Fish>(SARDINE, HERRING));
+
+    companion object {
+        private val FISHING_SPOTS = FishingSpot.values()
+
+        fun getFishingSpots(): Array<FishingSpot> = FISHING_SPOTS;
+    }
+}
+
+data class FishNpc(val npc: Int, val spot: FishingSpot, val locations: ArrayList<Position>) {
```

Comment:
I'd like if you merged this class with `FishingSpot` - rename `FishNpc` to `FishingSpot`, the existing `FishingSpot` to `Type`, and nest it inside this one.